app/org/sagebionetworks/bridge/validators/IdentifierUpdateValidator.java

```diff
@@ -39,13 +39,15 @@ public void validate(Object object, Errors errors) {
         if (update.getEmailUpdate() != null) {
             updateFields++;
         }
+        if (update.getExternalIdUpdate() != null) {
+            updateFields++;
+        }
```

Comment:
Should we add a unit test for valid ExternalId update? (Test is also missing valid phone update.)

Reply:
added tests for both these cases